server/app/views/applicant/ApplicantLayout.java

```diff
@@ -183,7 +183,12 @@ private DivTag maybeRenderTiButton(Optional<CiviFormProfile> profile, String use
       String tiDashboardText = "Trusted intermediary dashboard";
       String tiDashboardLink =
           controllers.ti.routes.TrustedIntermediaryController.dashboard(
-                  Optional.empty(), Optional.empty())
+                  /* paramName=  nameQuery */
+                  Optional.empty(),
+                  /* paramName=  searchDate */
+                  Optional.empty(),
+                  /* paramName=  page */
+                  Optional.empty())
```

Comment:
suggestion (blocking): fix comments to match the style used elsewhere in the codebase